prometheus/base/alerting/prometheus.controller_alerts.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    openshift.io/prometheus-rule-evaluation-scope: leaf-prometheus
+  name: o11y-controller-alerting-rules
+  namespace: o11y
+spec:
+  groups:
+  - name: controller_alerts
+    rules:
+    - alert: ControllerReconciliationErrors
+      expr: |
+        sum(increase(controller_runtime_reconcile_errors_total{namespace!~"(.*-tenant|openshift-.*|kube-.*|default)"}[2m]))
+        by(container, controller, namespace, pod, service) /
+        sum(increase(controller_runtime_reconcile_total{namespace!~"(.*-tenant|openshift-.*|kube-.*|default)"}[2m]))
+        by(container, controller, namespace, pod, service) > 0.01
```

Comment:
Do we actually need to `sum` here? Are there multiple metrics that share the same combination of `(container, controller, namespace, pod, service)`?

Reply:
`controller_runtime_reconcile_errors_total` is unique but `controller_runtime_reconcile_total` has multiple entries with different values in result field.

Reply:
different values that share the same combination of `(container, controller, namespace, pod, service)`? What's differentiating between those entries?

Reply:
Yes, only `result` and `value` changes for that combination for metric `controller_runtime_reconcile_total`

Reply:
alright. Thanks!